SUIApp/SUIApp/ContentView.swift

```diff
@@ -7,15 +7,148 @@
 
 import SwiftUI
 
+/// Content View
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let onOffStates = ["On", "Off"]
+        static let conectedStates =  ["Not Connected", "Connected"]
+        static let wifiStates = ["5G", "2G"]
+        static let personImageSystemName = "person.fill"
+        static let airplaneImageSystemName = "airplane"
+        static let wifiImageSystemName = "wifi"
+        static let bonjourImageSystemName = "bonjour"
+        static let mobileDataImageSystemName = "antenna.radiowaves.left.and.right"
+        static let vpnImageSystemName = "network.badge.shield.half.filled"
+        static let ImageSystemName = ""
+        static let navigationTitleText = "Settings"
+        static let userNameText = "Владислав Лымарь"
+        static let subTitleText = "Apple ID, iCloud+, Media&Purchases"
+        static let adsAppleText = "Apple TV+: 3 месяца бесплатно"
+        static let airplaneTitleText = "Airplane Mode"
+        static let wifiTitleText = "Wi-Fi"
+        static let bluetoothTitleText = "Bluetooth"
+        static let mobileDataTitleText = "Mobile Data"
+        static let vpnTitleText = "VPN"
+        static let cornerRadiusNumber: CGFloat = 5
+        static let defaultImageSizeNumber: CGFloat = 25
+        
+    }
+    
+    // MARK: - Private Property
+    
+    @State private var isOnAirplane = false
```

Comment:
```suggestion
    @State private var isOnAirplane = false
```
Все прайветы ниже var body